src/lib.rs

```diff
@@ -159,11 +159,11 @@
 //! its `stderr` output should warn about an unused variable on line 12; and the resulting binary
 //! should succeed produce `Hello world` on `stdout`.
 //!
-//! A file's tests can be ignored entirely if a test command `ignore` is defined:
+//! A file's tests can be ignored entirely with:
 //!
-//!   * `ignore: [<string>]` specifies that this file should be ignored for the reason set out in
-//!     `<string>` (if any). Note that `<string>` is purely for user information and has no effect
-//!     on the running of tests.
+//!   * `ignore-if-cmd: <cmd>` defines a shell command that will be run to determine whether to
```

Comment:
Should `ignore-if-cmd` be `ignore-if` or are they two separate facilities?

Reply:
Oops, that's a typo from an earlier design. Fixed in 679ac45.